app/src/types/ItemCardDataType.ts

```diff
@@ -0,0 +1,9 @@
+export interface ItemCardDataType {
```

Comment:
This is not entirely relevant to this PR but I found `CollectionCardDataType` and `CollectionDataType` confusing. They're very similar in name and properties. Likewise, `ItemCardDataType` and `CollectionCardDataType` share almost all properties. I think this one, `ItemCardDataType`, can be used as a base that `CollectionCardDataType` extends from with its own two additional properties.